include/aws/io/io.h

```diff
@@ -271,9 +254,26 @@ enum aws_io_errors {
     AWS_IO_STREAM_GET_LENGTH_FAILED,
     AWS_IO_STREAM_SEEK_UNSUPPORTED,
     AWS_IO_STREAM_GET_LENGTH_UNSUPPORTED,
+    AWS_IO_TLS_ERROR_READ_FAILURE,
 
     AWS_ERROR_PEM_MALFORMED,
 
+    AWS_IO_SOCKET_MISSING_EVENT_LOOP,
+    AWS_IO_TLS_UNKNOWN_ROOT_CERTIFICATE,
+    AWS_IO_TLS_NO_ROOT_CERTIFICATE_FOUND,
+    AWS_IO_TLS_CERTIFICATE_EXPIRED,
+    AWS_IO_TLS_CERTIFICATE_NOT_YET_VALID,
+    AWS_IO_TLS_BAD_CERTIFICATE,
+    AWS_IO_TLS_PEER_CERTIFICATE_EXPIRED,
+    AWS_IO_TLS_BAD_PEER_CERTIFICATE,
+    AWS_IO_TLS_PEER_CERTIFICATE_REVOKED,
+    AWS_IO_TLS_PEER_CERTIFICATE_UNKNOWN,
+    AWS_IO_TLS_INTERNAL_ERROR,
+    AWS_IO_TLS_CLOSED_GRACEFUL,
+    AWS_IO_TLS_CLOSED_ABORT,
+    AWS_IO_TLS_INVALID_CERTIFICATE_CHAIN,
+    AWS_IO_TLS_HOST_NAME_MISSMATCH,
```

Comment:
Not sure if it's too late. But it's misspelled 👗🧨
```suggestion
    AWS_IO_TLS_HOST_NAME_MISMATCH,
```

Reply:
It's a new one and should only be thrown at this point with SecItem. Fixed spelling.